production/technical_userguide.md

```diff
@@ -236,74 +236,89 @@ Metadata about the Beneficiary Claims Data API is available as a FHIR [Capabilit
+* **[Coverage](https://www.hl7.org/fhir/coverage.html){:target="_blank"}:** the information in this Resource Type indicates the beneficiary’s enrollment record. It includes information on a beneficiary’s Part A, Part B, Part C, and Part D coverage, as well as markers for End-stage Renal Disease (ESRD) and Old Age and Survivors Insurance (OASI).
 
-You can make a request to the `Patient` endpoint for all three Resource Types at once, one at a time, or a combination of any two together. This section describes a request for all three Resource Types; information about requesting one at a time follows below.
+#### The BCDA/bulk FHIR Endpoints
+- **Patient Endpoint:** Request `ExplanationOfBenefit`, `Patient`, and/or `Coverage` data for all beneficiaries. Filter data by date using the `_since` parameter.
+- **Group Endpoint:** Request `ExplanationOfBenefit`, `Patient`, and/or `Coverage` data for all beneficiaries. Filter data by date using the `_since` parameter.
+  - **Note: If you filter data using _since within the /Group endpoint, you will receive claims data since the date of your choice for existing beneficiaries AND you will also receive 7 years of historical data for all beneficiaries that are newly attributed to your ACO. Learn more about how to retrieve the most current beneficiary claims data for existing beneficiaries AND historical data for newly attributed beneficiaries.** 
 
-#### 1. Obtain an access token
+#### Requesting Current and Historical Data for all Beneficiaries
+You can make a request to the endpoints for all three Resource Types at once, one at a time, or a combination of any two together by specifying the resource type of interest (separated by commas) in the `_type` query parameter for both the `/Patient` and `/Group` endpoints. 
 
-See [Authentication and Authorization](#authentication-and-authorization) above.
+In the example below, we have provided `cURL` statements to retrieve any combination of the three resource types from the `/Patient` endpoint. Relevant `cURL` statements can be altered to retrieve any combination of resource types from the `/Group` endpoint as needed.
 
-#### 2. Initiate an export job
+##### 1. Obtain an access token
 
-**Request**
+See [Authentication and Authorization](https://bcda.cms.gov/production/technical-user-guide/#authentication-and-authorization){:target="_blank"} above.
+
+##### 2. Initiate an export job
+
+**Request all three resource types from the `/Patient` endpoint.** 
 
 `GET /api/v1/Patient/$export`
 
-To start a data export job for all three Resource Types, a GET request is made to the Patient endpoint. An access token as well as *Accept* and *Prefer* headers are required.
+**Request any two resource types from the `/Patient` endpoint.**
+
+`GET /api/v1/Patient/$export?_type=ExplanationOfBenefit,Patient`
 
+**Request a single resource type from the /Patient endpoint.**
+
+`GET /api/v1/Patient/$export?_type=ExplanationOfBenefit
```

Comment:
This doesn't render properly when running the site locally.  See screenshot:
<img width="644" alt="Screen Shot 2020-08-01 at 4 04 20 PM" src="https://user-images.githubusercontent.com/37818548/89109598-be919a80-d410-11ea-8116-8f9b2b420a48.png">

Reply:
Made each into its own line and separated them.